pkg/osquery/table/launcher_info.go

```diff
@@ -49,6 +61,15 @@ func generateLauncherInfoTable(db *bbolt.DB) table.GenerateFunc {
 			fingerprint = ""
 		}
 
+		if runtime.GOOS == "windows" || runtime.GOOS == "linux" {
```

Comment:
Does it make sense to leave it like this, or move the `setHardwareKeys` to a arch specific file.

Actually, I wonder if we need to push this all the way up to main. How do we handle machines _without_ a TPM? I'm sorta imagining that instead of the place we create RSA keys, we create an interface to the TPM/Secure Enclave/Local Storage and then pass it through somewhere

Reply:
Similarly, I think we should expose a column for what the source of the keys are.